providers/org/iacr/harvester.py

```diff
@@ -0,0 +1,38 @@
+import logging
+
+from lxml import etree
+from share import Harvester
+
+logger = logging.getLogger(__name__)
+
+
+class IACRHarvester(Harvester):
+
+    namespaces = {
+        'ns0': 'http://www.w3.org/2005/Atom'
+    }
+
+    url = 'https://eprint.iacr.org/rss/rss.xml'
+
+    def do_harvest(self, start_date, end_date):
+        # IACR has previous-search, you can only go from some past day to today
+        return self.fetch_records(self.url, start_date)
+
+    def fetch_records(self, url, start_date):
+        return self.fetch_page(url)
+
+    def fetch_page(self, url):
+        logger.info('Making request to {}'.format(url))
+
+        resp = self.requests.get(url, verify=False)
+        parsed = etree.fromstring(resp.content)
+        total_records = int(parsed.xpath("//ttl")[0].text)
+
+        records = parsed.xpath('//item', namespaces=self.namespaces)
+        logger.info('Found {} records of {}.'.format(len(records), total_records))
+        i = 0
+        for record in records:
+            doc_id = record.xpath('//guid', namespaces=self.namespaces)[i].text
```

Comment:
Yeah, I tested it and it worked. I think the problem was that XPath put all of the guid's into a single list so I was only ever getting the first one on that page rather than all the ones for each article.